test/evergreen/find_gperftools.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+# Currently evergreen machine doesn't contain gperftools binaries. The aim of this script is to
+# create a caching mechanism for fetching gperftools version 2.9.1. First we check if the binaries
+# for the machine are present in 10gen S3 bucket. If not, we download the sources, build the
+# binaries and upload the binaries to 10gen S3 Bucket.
+#
+# Note: The specific version number of gperftools 2.9.1 follows what mongodb currently uses. 
+#
+set +o verbose
+set +o errexit
```

Comment:
I think verbose and errexit are not turned on by default, which means these 2 lines (masking both options) have no actual effect?

Reply:
This here turns off verbose and exit on error. Turning off verbose is to ensure that we don't leak any output from this file. Turning off exit on error is necessary for this command `aws s3 ls "s3://build_external/jiechenbo_build/tcmalloc_${build_variant}.tgz"`. We want it to error, if it can't find the tcmalloc binaries.

Reply:
Evergreen shell command has a `silent` argument one can turn on to achieve the silence of shell output. Check out the [doc-build task](https://github.com/wiredtiger/wiredtiger/blob/0c90aafd226dbf337f294bd4b3beb26060c1109e/test/evergreen.yml#L296) as an example.

Reply:
Hmm, correct me if I'm wrong but it looks like that command silences the output of the entire function/task? (unless you can manually override the `silent` argument mid-function?)
I'm not sure that would be ideal since the `find_gperftools.sh` script is run inside the `configure wiredtiger` function.  We definitely wouldn't want to make that whole function silent as the output of the build configure task is valuable to log (i.e. checking the configuration step ran correctly).